Live/Classes/HUD/HUDBar.h

```diff
@@ -0,0 +1,36 @@
+#ifndef _HUD_BAR_
+#define _HUD_BAR_
```

Comment:
don't use header guards, use `#pragma once` instead, we are switching to that in other files too, slowly.